manifests/aqua/0.0.1/aquagateways.operator.aquasec.com.crd.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Rather than add something to the root of the `manifests/` folder, can we just add these to a `test_data` folder for a separate test?

Reply:
awww, I changed all those unit tests for it. Shouda thought about it... making the change now